docs/overview.md

```diff
@@ -1,6 +1,35 @@
-# Overview
+page_title: Registry Service 2.0
+page_description: Explains how to deploy a registry service
+page_keywords: registry, service, images, repository
 
-**TODO(stevvooe):** Table of contents.
+# Registry Service 2.0
 
-**TODO(stevvooe):** Include a full overview of each component and dispatch the
-user to the correct documentation.
+**TODO(stevvooe):** Include a full overview of each component and dispatch the user to the correct documentation.
+
+
+
+### Upgrading
+
+The Docker Registry Service 2.0 is backward compatible with images created by the 1.0 specification. If you are migrating a private registry to version 2.0, you should use the following process:
```

Comment:
Y'all, if a section has no content and just  TODO we should just remove the section. Do you want to go out the door without a migration path for 1.0 users? (BTW, I just took a stab at it here in hopes someone would correct me so thanks for that. :)